mongomock/database.py

```diff
@@ -16,6 +18,8 @@
 except ImportError:
     _READ_PREFERENCE_PRIMARY = read_preferences.PRIMARY
 
+list_collection_filter_allowed_operators = frozenset(['$regex', '$eq', '$ne'])
```

Comment:
Please name that _LIST_COLLECTION_FILTER_ALLOWED_OPERATORS (upper case for module constants, prefixed by `_` as it's supposed to be used in this module only).